client/new-stack-editor/lib/views/sideview.coffee

```diff
@@ -50,17 +52,43 @@ module.exports = class SideView extends BaseView
     for _view, item of @getOption 'views'
 
       if _view is viewName
-        @title.updatePartial item.title
+        if item.title?
+          @title.updatePartial item.title
+          @title.show()
+        else
+          @title.hide()
         @setClass item.cssClass
         item.view.show()
 
       else
         @unsetClass item.cssClass
         item.view.hide()
 
+    { expanded, collapsed = yes } = options
```

Comment:
show as is (with the prev state) `unless options`
show expanded `if options.expanded`
show collapsed `unless options.expanded`